spatialpandas/dask.py

```diff
@@ -237,6 +237,8 @@ def pack_partitions_to_parquet(
 
                 These directories are deleted as soon as possible during the execution
                 of the function.
+            storage_options: Key/value pairs to be passed on to the file-system backend, if any.
+            engine_kwargs: pyarrow.parquet engine-related keyword arguments. 
```

Comment:
Need to add something like `engine_kwargs = engine_kwargs or {}` near the top of the function.

Reply:
Done!